terratorch/tasks/segmentation_tasks.py

```diff
@@ -121,9 +136,16 @@ def configure_callbacks(self) -> list[Callback]:
         return []
 
     def configure_models(self) -> None:
+        if not hasattr(self, "model_factory"):
+            # Custom model is provided
+            if self.hparams["freeze_backbone"] or self.hparams["freeze_decoder"]:
+                logger.warning("freeze_backbone and freeze_decoder are ignored if a custom model is provided.")
+            return
```

Comment:
@Joao-L-S-Almeida see same problem as above

Reply:
The code is okay.